src/index.js

```diff
@@ -6,13 +6,19 @@ import state from "./state";
 
 function render() {
```

Comment:
we don't need to put everything in a render function
only the code on line 16 to 23
otherwise how will the category ever change